File: 1 - metodosBusca/pacoteBusca_JomiHubner/src/grafo/GrafoNaoDirigido.java

package grafo;

public class GrafoNaoDirigido extends Grafo {
	public void criaAresta(int i, int j) {
		Vertice vi = vertices.get(i);
		Vertice vj = vertices.get(j);
		new ArestaNaoDirigida(vi,vj);
	}

    public void criaAresta(int i, int j, int custo) {
        Vertice vi = vertices.get(i);
        Vertice vj = vertices.get(j);
        new ArestaNaoDirigidaValorada(vi,vj,custo);
    }
}

File: 1 - metodosBusca/python/app/core/grafo/grafo_nao_dirigido.py
from .grafo import Grafo
from .aresta_nao_dirigida import ArestaNaoDirigida
from .aresta_nao_dirigida_valorada import ArestaNaoDirigidaValorada

class GrafoNaoDirigido(Grafo):
    def cria_aresta(self, i, j):
        vi = self.get_vertice(i)
        vj = self.get_vertice(j)
        ArestaNaoDirigida(vi, vj)

    def cria_aresta(self, i, j, custo):
        vi = self.get_vertice(i)
        vj = self.get_vertice(j)
        ArestaNaoDirigidaValorada(vi, vj, custo)